spec/models/loader_orchestrate_spec.rb

```diff
@@ -16,7 +16,7 @@ def load_base_policy path
     RootLoader.load 'rspec', policy_path(path)
   end
 
-  def policy_version path
+  def save_policy path
```

Comment:
Add parens :)

Might as well update the rest in this file since you're touching it.